sound/soc/sof/ipc3-loader.c

```diff
@@ -131,6 +131,8 @@ static ssize_t ipc3_fw_ext_man_size(const struct firmware *fw)
 		return head->full_size;
 
 	/* otherwise given fw don't have an extended manifest */
+	dev_dbg(sdev->dev, "Missing or invalid ext manifest magic number: %#x\n",
```

Comment:
why not just say "Unexpected manifest number: %#x\n"...). If its missing, how can we even print it?

Reply:
I carbon copied the ipc4 mesaage, where we actually fail as thw manifest is mandatory there.
Should we change the ipc4 print to 'unexpected' version at the same time (before I send it upstream)?

Reply:
if it makes sense, yes @ujfalusi